### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by DemiMarie** [servo/servo#27672] [General] [answered]
Is Servo still alive after the Mozilla layoffs?

**Comment by jdm** [comment, marked as answer]
The project still exists and the maintainers still have the ability to merge changes, and some changes have even been merged quite recently.

**Comment by DemiMarie** [comment]
Is it still being actively developed?  Will it continue to be developed in the future?

**Reply by niutech** [reply]
As you can see in the [commit history](https://github.com/servo/servo/commits/master), it is still being actively developed.
